chaosaws/elbv2/actions.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+import random
+from collections import Counter
+from copy import deepcopy
+from typing import Any, Dict, List
+
+import boto3
+from logzero import logger
+
+from chaosaws import aws_client
+from chaosaws.types import AWSResponse
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+
+__all__ = ["deregister_target"]
+
+
+def deregister_target(tg_name: str,
+                      configuration: Configuration = None,
+                      secrets: Secrets = None) -> AWSResponse:
+    """
+    Deregisters one random target from target group
+    """
+    client = aws_client('elbv2', configuration, secrets)
+    tg_arn = get_target_group_arns(tg_names=[tg_name], client=client)
+    tg_health = get_targets_health_description(tg_arns=tg_arn, client=client)
+    random_target = random.choice(tg_health[tg_name]
+                                  ['TargetHealthDescriptions'])['Target']['Id']
+    logger.debug("Deregistering target {} from target group {}".format(
+        random_target, tg_name))
+
+    return client.deregister_targets(TargetGroupArn=tg_arn[tg_name],
+                                     Targets=[{'Id': random_target}])
+
+###############################################################################
+# Private functions
+###############################################################################
+
+
+def get_target_group_arns(tg_names: List[str],
```

Comment:
Only used by the above action function at the moment, but I'm wondering if this and the following function could in fact be probes?

Reply:
Let's stick with this for now, we can refactor and refine going forward if necessary.